pkg/jobrunaggregator/jobrunbigqueryloader/test_run_uploader.go

```diff
@@ -51,13 +50,33 @@ func (o *testRunUploader) uploadTestSuite(ctx context.Context, jobRun jobrunaggr
 		}
 	}
 
-	toInsert := []bigquery.ValueSaver{}
+	toInsert := []*jobrunaggregatorapi.TestRunRow{}
 	for i := range suite.TestCases {
 		testCase := suite.TestCases[i]
 		if testCase.SkipMessage != nil {
 			continue
 		}
-		toInsert = append(toInsert, newTestRunRow(jobRun, prowJob, currSuites, testCase))
+
+		var status string
+		switch {
+		case testCase.FailureOutput != nil:
+			status = "Failed"
+		case testCase.SkipMessage != nil:
+			status = "Skipped"
+		default:
+			status = "Passed"
+		}
+
+		testSuiteStr := strings.Join(currSuites, "|||")
+		if suite.Children != nil {
+			// There is generally a single test suite.  But if there are more, we will
+			// concat them to make a unique test suite name.
+			for _, childTestSuite := range suite.Children {
```

Comment:
why do we add the children of this test suite to the test case.  it logically looks like

```yaml
suiteName: foo
- childSuiteName: bar
  - testCaseName: the-test
  - childSuiteName: trip
    - testCaseName:  smaller-test
  - childSuiteName: fourth
    - testCaseName:  now-a-test
```

we want
1. foo|||bar the-test
2. foo|||bar|||trip smaller-test
3. foo|||bar|||four now-a-test

but this code gives me
1. foo||bar||trip||four the-test
2. foo|||bar|||trip smaller-test
3. foo|||bar|||four now-a-test